content/common/utils/utils.h

```diff
@@ -0,0 +1,49 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#ifndef UTILS_H_
+#define UTILS_H_ 1
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#if defined(__linux__) || defined(__Unikraft__)
+#define DIE(assertion, call_description)			\
+	do {							\
+		if (assertion) {				\
+			fprintf(stderr, "(%s, %d): ",		\
+					__FILE__, __LINE__);	\
+			perror(call_description);		\
+			exit(EXIT_FAILURE);			\
+		}						\
+	} while (0)
+
+#elif defined(_WIN32)
```

Comment:
I think this should go for Windows on 64 bits as well, right?

Reply:
`_WIN32` is enabled on both 32-bit and 64-bit systems. `_WIN64` is only enabled on 64-bit systems [1]. So `_WIN32` covers both platforms.

[1] https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170